actix-web-actors/tests/test_ws.rs

```diff
@@ -1,8 +1,9 @@
 use actix::prelude::*;
-use actix_web::{test, web, App, HttpRequest};
+use actix_web::{test, web, App, HttpRequest, HttpResponse};
 use actix_web_actors::*;
 use bytes::Bytes;
 use futures_util::{SinkExt, StreamExt};
+use http::StatusCode;
```

Comment:
status code is re-exported though actix_web::http::StatusCode and awc::http::StatusCode so extra dependency in cargo.toml is not needed